agent/php_user_instrument.c

```diff
@@ -510,7 +518,8 @@ void nr_php_add_user_instrumentation(TSRMLS_D) {
 void nr_php_add_transaction_naming_function(const char* namestr,
                                             int namestrlen TSRMLS_DC) {
   nruserfn_t* wraprec
-      = nr_php_add_custom_tracer_named(namestr, namestrlen TSRMLS_CC);
+      = nr_php_add_custom_tracer_named(namestr, namestrlen,
+                                       false /*is_transient*/ TSRMLS_CC);
```

Comment:
Instead of adding `/*is_transient*/` as a comment in the function call, why not create a `bool is_transient = false` and pass the new `is_transient` variable instead?

Reply:
Mainly as a standard. Creating a local variable is not very extendable if, say, 5 variables are passed as constants. Additionally, this doesn't clutter the local variable space (less chance of accidentally passing the same variable to a different function with the wrong value), protects this call from a similar affect, and I find it easier to read.

Reply:
If only C had named function arguments...

Reply:
I think [this suggestion](https://github.com/newrelic/newrelic-php-agent/pull/679/files#r1228571233) will make this change unnecessary.

Reply:
refactored in [ea328b5](https://github.com/newrelic/newrelic-php-agent/pull/679/commits/ea328b5b408d4efa889f4e6bc9d2733cca2301e2)